charts/thoras/templates/crd/aiscaletarget.yaml

```diff
@@ -105,8 +105,28 @@ spec:
                         - type: integer
                         - type: string
                       x-kubernetes-int-or-string: true
-                    forecast_buffer_percentage:
-                      x-kubernetes-int-or-string: true
+                reasoning:
+                  type: object
+                  required:
+                    - metrics
+                  properties:
+                    metrics:
+                      type: array
+                      items:
+                        type: object
+                        required:
+                            - type
+                            - name
+                            - query
+                        properties:
+                          type:
+                            type: string
+                            pattern: ^(prom)$
```

Comment:
These regexes not being in quotes scares me a bit given how YAML has a specific semantics around certain symbols -- would it hurt to throw quotes around these pattern fields?